cmd/cli/cmd/alpha/manifest-gen/implementation/questions.go

```diff
@@ -0,0 +1,119 @@
+package implementation
+
+import (
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/common"
+	"capact.io/capact/internal/cli/alpha/manifestgen"
+	"capact.io/capact/pkg/sdk/apis/0.0.1/types"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+)
+
+type helmChart struct {
```

Comment:
this is not necessary - if you ask about name here (as described in other comment) then you can use `helm.Chart` type from `pkg/runner/helm/types.go` 🙂